portfolio/src/main/webapp/index.html

```diff
@@ -46,7 +46,59 @@ <h2>About Me </h2>
 <div id="timelapse">
   <h2>Timelapses</h2>
   <p>
-    Here are some videos that I have taken and edited
+    Here are some videos that I have taken and edited.
+  <div id="timelapse-slideshow" class="carousel slide" data-ride="carousel">
+    <ol class="carousel-indicators">
+      <li data-target="#timelapse-slideshow" data-slide-to="0" class="active"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="1"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="3"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="4"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="5"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="6"></li>
+    </ol>
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/1.mov" type="video/mp4" />
```

Comment:
Something to keep in mind when including multimedia - it's nice to have a fallback or alternative - like text representing the content. This is especially helpful for accessibility needs. 
https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

You don't need to add a fallback for each vid unless you'd like, but more of an FYI  :)

Reply:
Added a fallback as in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video

Reply:
Nit: remove optional closing mark `/`

```html
<source src="resources/videos/1.mov" type="video/mp4">
```

Reply:
Made Changes